Automattic/themes
# Seedlet: support filtering archive page titles

This PR addresses #2317.

**To test**
- Activate seedlet and visit any page that uses the archive template
- Verify that the prefix is wrapped in a span with the CSS class `archive-prefix`
